vip-helpers/vip-newrelic.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * Disable New Relic's browser metrics
+ *
+ * Removes NR's JavaScript for tracking browser metrics, including page load times, Apdex score, and more.
+ *
```

Comment:
That's how it was previously, I think in code is easier for devs to do since they might want to remove it but not have access to New Relic.